src/sections/image-with-text.liquid

```diff
@@ -0,0 +1,110 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add "Image With Text" section on the Theme Editor.
```

Comment:
Corrections in bold:

1. **the** "Image **w**ith **t**ext" section
2. **in** the **t**heme **e**ditor

Reply:
Good eyes, thanks!
I learned the capitalization tip during the content strategy workshop at RnD camp but it totally slipped my mind 🤦‍♂️